worker/src/unstract/worker/clients/docker.py

```diff
@@ -168,18 +168,20 @@ def get_container_run_config(
             envs = {}
         mounts = []
         if organization_id and workflow_id and execution_id:
+            source_path = os.path.join(
+                os.getenv(Env.WORKFLOW_DATA_DIR, ""),
+                organization_id,
+                workflow_id,
+                execution_id,
+            )
             mounts.append(
                 {
                     "type": "bind",
-                    "source": os.path.join(
-                        os.getenv(Env.WORKFLOW_DATA_DIR, ""),
-                        organization_id,
-                        workflow_id,
-                        execution_id,
-                    ),
+                    "source": source_path,
                     "target": os.getenv(Env.TOOL_DATA_DIR, "/data"),
                 }
             )
+            envs[Env.WORKFLOW_DATA_DIR] = source_path
```

Comment:
@Deepak-Kesavan why are we adding this here?

Reply:
@ritwik-g It is required for table extractor to work in structure tool.

Reply:
@Deepak-Kesavan can you change the ENV name?

Reply:
Done